fase4_fsm:
  ros__parameters:
    # Home position parameter
    fictual_home_x: 0.0
    fictual_home_y: 0.0
    fictual_home_z: 0.0 # Remember z points Down

    # Aruco alignment parameters
    timeout_aruco_align: 5.0
    normalized_position_tolerance: 0.1

    # Takeoff parameters
    takeoff_height: -1.5
    max_vertical_velocity: 0.5
    
    # Landing parameters
    landing_velocity: 0.3
    landing_timeout: 10.0
    
    # Moviment params
    max_horizontal_velocity: 0.5
    max_yaw_rate: 1.0
    position_tolerance: 0.1
    
    # PID parameters for positioning (valores mais conservadores)
    pid_pos_kp: 2.0
    pid_pos_ki: 0.0
    pid_pos_kd: 0.5
    setpoint: 0.0

fase4_vision:
  ros__parameters:
    timeout: 10.0